source/Server/Decompilation/ILDecompiler.cs

```diff
@@ -23,6 +23,13 @@ public class ILDecompiler : IDecompiler {
             Argument.NotNull(nameof(streams), streams);
             Argument.NotNull(nameof(codeWriter), codeWriter);
 
+            // Write header
+            codeWriter.WriteLine(
+@".assembly SharpLabAssembly
+{
+}
+");

```

Comment:
Is there any way (setting) to convince `PEFile` that's it's an assembly, even though we give it a stream?
If not that's fine, but feels a bit surprising we have to write it manually.

Reply:
Now that you mentioned it, I took a deeper look. It turned out setting name (first argument), `"_"` did the trick. :)